features/admin.identity-providers.v1/components/forms/authenticators/fido-authenticator-form.tsx

```diff
@@ -207,46 +204,44 @@ export const FIDOAuthenticatorForm: FunctionComponent<FIDOAuthenticatorFormProps
 
         updateFidoConfigs(payload)
             .then(() => {
-                dispatch(addAlert({
+                addAlert({
```

Comment:
These dispatch function should be there if we are keeping addAlert function. Otherwise it will not display an error/success popup. As I remember previously it was not there and I added it with the trusted app UI. So, let's keep it.